tests/test_banneduser_experiment_controller.py

```diff
@@ -271,6 +271,20 @@ def test_is_tempban(self, action, details, want, experiment_controller):
         got = experiment_controller._is_tempban({"action": action, "details": details})
         assert got == want
 
+    @pytest.mark.parametrize(
+        "action,details,want",
+        [
+            ("banuser", "3 days", False),
+            ("banuser", "permanent", False),
+            ("removecomment", "remove", False),
+            ("banuser", "changed to 1 days", True),
+            ("banuser", "changed to 3 days", True),
+        ],
+    )
+    def test_is_tempban_edit(self, action, details, want, experiment_controller):
+        got = experiment_controller._is_tempban_edit({"action": action, "details": details})
```

Comment:
(will work after merging #83)

```suggestion
        got = experiment_controller._is_tempban_edit(DictObject({"action": action, "details": details}))
```